src/engraving/libmscore/textbase.cpp

```diff
@@ -3365,7 +3384,8 @@ void TextBase::drawEditMode(mu::draw::Painter* p, EditData& ed)
     p->setPen(pen);
 
     // Don't draw cursor if there is a selection
-    if (!cursor->hasSelection()) {
+    LOGD() << _hideCursor;
+    if (!cursor->hasSelection() && !_hideCursor) {
```

Comment:
don't think we need that log message - what's an example of why this would be wanted on a per-element basis?